gulpfile.js

```diff
@@ -19,7 +19,7 @@ gulp.task('clean', function() {
 });
 
 gulp.task('lint', () => {
-  return gulp.src(['src/*.gs', 'samples/*.gs', 'test/**/*.js', '!node_modules/**'])
+  return gulp.src(['src/*.js', 'samples/*.gs', 'test/**/*.js', '!node_modules/**'])
```

Comment:
At the moment it wouldn't be possible to use clasp to deploy the samples, since they are all in the same directory and have duplicate method names. Once we've finalized some file naming conventions I'll do a pass to standardize the library.